tbx/users/tests/test_models.py

```diff
@@ -0,0 +1,8 @@
+from django.test import TestCase
+
+from tbx.users.factories import UserFactory
+
+
+class TestUserFactory(TestCase):
+    def test_create(self):
+        UserFactory()
```

Comment:
@engineervix I can see the value in this test if we plan to use the `UserFactory` a lot elsewhere (e.g. to test functionality as different users), but from what I can tell, we're not doing that. Let's just be sure that when adding tests in future, they are for something worthy of testing, and not just to bump up coverage.